raspberrypi-node.js

```diff
@@ -26,7 +26,6 @@ function initTasks(gulp, options) {
     var files = fs.readdirSync('./app');
     var filesLocal = [];
     var filesRemote = [];
-    console.log(files);
 
     for (var i = 0; i < files.length; i++) {
       filesLocal.push('./app/' + files[i]);
```

Comment:
File config.json won't be logged due to below two lines. It seems buggy. So, simply remove the console.log

     filesLocal.push('./config.json');
     filesRemote.push(targetFolder + '/config.json');